app/script/event/EventRepository.coffee

```diff
@@ -338,7 +338,7 @@ class z.event.EventRepository
       if sending_client
         log_message = "Received encrypted event '#{event.type}' from client '#{sending_client}' of user '#{event.from}'"
       else if event.from
-        log_message = "Received plain event '#{event.id}' of type '#{event.type}' from client '#{sending_client}' of user '#{event.from}'"
+        log_message = "Received unencrypted event '#{event.id}' of type '#{event.type}' from user '#{event.from}'"
```

Comment:
Sending clients do not exist for unencrypted messages. It will always show undefined in the log.